src/components/PrintButton/PrintButton.js

```diff
@@ -23,26 +25,39 @@ class PrintButton extends React.Component {
     onBeforeGetContent: noop,
   };
 
+  eventObject = {};
+
   getContent = () => {
     return this.props.contentRef.current;
   };
 
+  handlePrintBeforeGetContent = () => {
+    if (this.eventObject.event !== undefined) {
+      this.eventObject.event.stopPropagation();
+    }
```

Comment:
Do we need `if` condition here?
I think for now it is redundant and we can write the following code:

```suggestion
      this.eventObject.event.stopPropagation();
```

Reply:
It was failing during testing. There are cases where event object is undefined and it thorws exception. So To safeguard it, it wrapped in if statement. 
Do you see a downside?

Reply:
Did you test it manually?
I run your changes locally and tested manually, looks like it works for me.